### ISSUE TITLE
'control-c/cmd-c' shortcuts should use 'editor:copy-selection' command by default

### ISSUE THREAD

**Issue by TimeTerminal** [atom/atom#18238]
## Summary

Currently the `control-c` (Windows/Linux) or `cmd-c` (Mac) shortcut follows the 'core:copy' command. This command essentially copies whatever text is selected in the editor. But it also selects empty line if nothing is selected.

## Motivation

This is especially a problem if you accidentally pressed the copy shortcut before selecting some text.

The proposal is to have the default behavior to use the 'editor:copy-selection' command. This way, it would only copy something to the clipboard if some text is highlighted. If nothing is highlighted, it will keep the last copied item in the clipboard memory.

And if someone needs to copy an empty line to paste it, they can just press the Enter/Return key

## Describe alternatives you've considered

This was previously suggested in #7103. It was fixed, but since then Atom updated and it broke it again.

**Comment by TimeTerminal** [comment]
#7103

**Comment by TimeTerminal** [comment]
#7339

**Comment by lee-dohm** [comment]
The current behavior [is intentional](https://github.com/atom/atom/pull/4148). If you want the <kbd>Cmd+C</kbd> or <kbd>Ctrl+C</kbd> behavior to use `editor:copy-selection`, you can change your personal keymap. We aren't going to be changing the default behavior.

**Comment by TimeTerminal** [comment]
If you look at #7103 a lot of people have shown interest in this feature. Sure you can go to the keymap and change it right now. Versus a few people who chose to have the `core:copy` command back in 2014 in #4148. And even that too with mixed support for it.

If Atom's supposed to be an open source, community driven project, shouldn't we at least discuss to see which one people would prefer to have as the default one? And the ability to have the empty line would still be retained of course.

**Comment by lee-dohm** [comment]
Unfortunately, issues aren't a good way to get a representative sample of the opinions of the Atom userbase. This is because people generally only go to issues when they have a question or concern. The people that are satisfied with the way things are won't be represented in the discussion because they have no motivation to go look at the issues list. We have learned this from repeated "discussions" where we only hear from the people that disagree, we change the behavior to comply with their ideas, then we only hear from the people that disagree with the new behavior. For these reasons, we've decided to have a _very_ high bar to change defaults. This does not meet that bar.
